srcpkgs/gpsbabel/template

```diff
@@ -1,23 +1,25 @@
 # Template file for 'gpsbabel'
 pkgname=gpsbabel
-version=1.5.4
+version=1.7.0
 revision=1
 wrksrc="gpsbabel-gpsbabel_${version//./_}"
 build_style=gnu-configure
 build_helper=qmake
-makedepends="qt5-devel libusb-compat-devel"
+makedepends="qt5-devel qt5-webkit-devel libusb-compat-devel"
 depends="desktop-file-utils"
 short_desc="Converts waypoints, tracks, and routes between popular GPS formats"
 maintainer="Philipp Hirsch <itself@hanspolo.net>"
 license="GPL-2.0-or-later"
 homepage="https://www.gpsbabel.org/"
 distfiles="https://github.com/gpsbabel/${pkgname}/archive/${pkgname}_${version//./_}.tar.gz"
-checksum=8cd740db0b92610abff71e942e8a987df58cd6ca5f25cca86e15f2b00e190704
+checksum=30b186631fb43db576b8177385ed5c31a5a15c02a6bc07bae1e0d7af9058a797
 
 replaces="gpsbabel-gui>=0"
 
+nocross="Configure cannot find Qt library."
+
```

Comment:
Adding nocross to a previously cross package is kinda bad :/

Reply:
Sure, but I was not able to make it work. Not sure if it really did work before or the error was just ignored. I assume the later. If someone can help that would be great.

Reply:
Fixed